konflux/README.md

```diff
@@ -0,0 +1,53 @@
+# Konflux integration test cases
+As RedHat uses Konflux to build, verify and relase containers now, all fido containers will be built in konflux. The integration test cases defined in this folder will be triggered by konflux when new fido container images are built.
+
+## How konflux trigger test cases
+konflux defines integration test workflow in this file: (when writting this doc, below files are not merged into konflux yet, will update it later)
+- https://gitlab.cee.redhat.com/releng/konflux-release-data/-/merge_requests/2167/diffs#3904e36f53d22f073f197dedd0bd6355289a0053
+
+Within this file, it defines fido repo url and test file path:
+- fido repo: https://gitlab.cee.redhat.com/releng/konflux-release-data/-/merge_requests/2167/diffs#3904e36f53d22f073f197dedd0bd6355289a0053_0_15
+- test file: https://gitlab.cee.redhat.com/releng/konflux-release-data/-/merge_requests/2167/diffs#3904e36f53d22f073f197dedd0bd6355289a0053_0_19
+
+When konflux trigger integration test workflow, it will go to fido repo and get the test file, and execute the tasks defined in test file.
+
+## Test files in this folder
+There are four pipeline files and one test file in this folder.
+- fdo-manufacturing-server-integration-tests.yaml
+- fdo-owner-onboarding-server-integration-tests.yaml
+- fdo-rendezvous-server-integration-tests.yaml
+- fdo-serviceinfo-api-server-integration-tests.yaml
+- fido-container-test.yaml
```

Comment:
Please rename to `fdo-container-test.yaml`.
The same applies to the rest of ocurrences in the document.

Reply:
fixed